ui/src/timespan/TimeSpan.tsx

```diff
@@ -16,14 +16,13 @@ import Menu from '@material-ui/core/Menu';
 import MenuItem from '@material-ui/core/MenuItem';
```

Comment:
> Selecting a timespan and pressing delete or backspace will delete it.
> Pressing escape will unselect.

> Pressing escape after creating a new timespan will delete the newly created timespan until it or another timespan has been selected. 

In my testing backspace/delete works too for newly created timespans (by dragging). Why does we also need to support escape here? I rather not have escape mean sometimes delete and sometimes deselect.

Reply:
It just feels really intuitive in my opinion and a more convenient key press for quick mistakes.